src/views/Player.vue

```diff
@@ -186,6 +190,22 @@ export default {
 
           // set the global list of time values
           this.times = res.data.times
+
+          // merge the previous session data
+          if (res.data.sessionData != null) {
+            this.answers = res.data.sessionData.answers
+            for (i=0; i < questions.length; i++){
+              this.plioQuestions[i].user_answer = this.answers[i]
+              this.plioQuestions[i].state = 
+                (this.answers[i].length == 0) ? "notshown" : "answered"
+            }
+            
+            this.journey = res.data.sessionData.journey
+            this.previousPlayerTime = (
+              (this.journey.length > 0) ? this.journey[this.journey.length - 1]['player_time'] : 0)
+            this.watchTime += res.data.sessionData['watch-time']
+            this.retention = res.data.sessionData.retention
+          }
```

Comment:
- copy the answers array
- copy the answers into `plioQuestions[x].user_answer` as well
- also set the state as `answered` or `notshown`
- copy the journey array
- set the player previous time which will be used later to set the current time
- add the watch time
- copy the retention